releases/geneva/compose-files/docker-compose-geneva-redis-no-secty-arm64.yml

```diff
@@ -235,16 +233,14 @@ services:
       - data
 
   rulesengine:
-    image: emqx/kuiper:0.3.2
+    image: emqx/kuiper:0.4.2-alpine-alpine
```

Comment:
emqx/kuiper:0.4.2-alpine not emqx/kuiper:0.4.2-alpine-alpine

Reply:
Good catch, by search and replaces must have been off a bit... ;-) Fixed, THX!